modules/layouts/LayoutCS.chpl

```diff
@@ -328,9 +330,9 @@ class CSDom: BaseSparseDomImpl {
 
   proc dsiFirst {
     if _nnz == 0 then return (parentDom.low) - (1,1);
-    const _low = nnzDom.low;
```

Comment:
This was probably a (premature?) optimization to capture the field in a variable before entering the loop. While I cannot be certain whether loop invariant code motion will be able to hoist  the corresponding access in the new version, I hardly think this is a performance issue. I just wanted to note it.